frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.scss

```diff
@@ -23,3 +23,10 @@
 .pf-c-wizard__toggle {
   z-index: calc(var(--pf-global--ZIndex--xs) - 1);
 }
+
+.pf-c-wizard__nav-item {
```

Comment:
So the reason why we aren't getting the right cursor is because the `<a>` doesn't have an `Href` attribute.

Reply:
Actually it's because it is being overridden by bootstarp

Reply:
in either case I think we'd want to do more than just add styles to account for the hover state. Because the link lacks the necessary href we are also missing out on the tabbing for keyboard users. 

I've followed up on an already open issue on the PatternFly React side here
https://github.com/patternfly/patternfly-react/issues/1588